src/components/Home/Landing/index.tsx

```diff
@@ -1,18 +1,64 @@
-import { FC } from 'react';
+import { FC, useState } from 'react';
 import NavBar from 'components/NavBar';
 import { useQuery } from '@tanstack/react-query';
 import config from 'config';
 import { getAccessToken } from 'utils/authFn';
 import BlurBlobs from 'components/BlurBlobs';
 import useFetchOffChainProfile from 'hooks/useFetchOffChainProfile';
+import Shepherd from 'shepherd.js';
+import { useNavigate } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import { RootState } from 'reducers';
 import Tasks from '../Tasks';
 import Banner from '../Banner';
 import Projects from '../Projects';
 import styles from './index.module.scss';
 
 const Landing: FC = () => {
   // const { offChainProfile } = useFetchOffChainProfile();
-
+  const navigate = useNavigate();
+  const profile = useSelector((state: RootState) => state.profile.profile);
+  const [checkOnboardStatus, setCheckOnboardStatus] = useState(
+    localStorage.getItem('onboardStatus')
```

Comment:
There's a `getItem` function in `localStorageFn.ts` file which can be used to get the value